src/ButtonInput.js

```diff
@@ -17,14 +17,14 @@ class ButtonInput extends InputBase {
   }
 }
 
-ButtonInput.types = ['button', 'reset', 'submit'];
+ButtonInput.types = Button.types;
```

Comment:
Thanks for the review. Happy to remove it. It was an effort on my part to preserve the 'interface' as such. If third parties are relying on it being there. I don't know whether you treat that as part of your interface or not.